### ISSUE TITLE
Donate Page for individual team

### ISSUE THREAD

**Issue by LaurenMcM749** [Amaxa-Impact/amaxa#31]
Problem: Fundraising teams currently use a variety of methods to collect funds. Not only is this painful for them, but these platform take a fee, AND we are missing opportunities for new people, like parents, commuinty members donating, to see who we are.

Solution: Each team has their own unique donate page. 

Priority: Very High

**Comment by not-ani** [comment]
We can definitely add ways to link to multiple different fundraising sources onto a single page for streamlining the process,  but I don't think we can avoid the fee, we would use something like stripe to process payments

**Comment by LaurenMcM749** [comment]
Yes that's fine. I believe Justgiving is the best portal we should start using.
